numba/tests/test_unicode.py

```diff
@@ -1701,7 +1701,12 @@ def test_strip(self):
             ('  tú quién te crees?   ', None),
             ('大处 着眼，小处着手。大大大处', '大处'),
             (' 大处大处  ', ''),
-            (' 大处大处  ', None)
+            (' 大处大处  ', None),
+            ('\t abcd \t', None),
+            ('\n abcd \n', None),
+            ('\r abcd \r', None),
+            ('\x0b abcd \x0b', None),
+            ('\x0c abcd \x0c', None)
```

Comment:
Perhaps also add a version of example here https://github.com/numba/numba/pull/5213#issuecomment-584062365 with a unicode kind 4 whitespace either side of some ASCII? e.g. `'\u2029abcd\u205F'` ?

Reply:
And also add in some variants where the `chars` kwarg is something other than `None`?

Reply:
I have added two more cases, please take a look.

Reply:
Done. Added a case where `chars` is something other than `None`